packages/playwright/src/reporter.ts

```diff
@@ -279,7 +278,7 @@ class ReplayPlaywrightReporter implements Reporter {
               try {
                 return [filename, readFileSync(filename, "utf8")];
               } catch (e) {
-                warn(`Failed to read playwright test source for: ${filename}`, e);
+                debug(`Failed to read playwright test source for: ${filename}`, e);
```

Comment:
This surprised me: do we expect this to happen often enough that we want to hide those warnings by default? I understand that we want to hide the annotation warnings by default because that happens so often but I've never seen this warning.

Reply:
This shouldn't happen often. However, we get those filenames from the call stacks - I'm not quite sure how reliable this is (especially considering source mapped sources etc). 

It could be good to raise those as errors somewhere... but perhaps this should be done by aggregating those errors and surfacing this after the whole run? Outputting this when it happens is pretty random and interferes with whatever reporters u have enabled. WDYT?

Reply:
OK, I wasn't aware that it can interfere with other reporters - in that case it makes sense to hide those messages by default.
We could raise these errors somewhere else but if we don't have a mechanism for doing so already I guess that would be a separate issue.